apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibility.jsx

```diff
@@ -226,25 +223,53 @@ export default class AmazonFutureEngineerEligibility extends React.Component {
   };
 
   loadCompletionPage = async () => {
-    let sessionEligibilityData = this.getSessionEligibilityData();
-
-    if (!sessionEligibilityData.submitted) {
-      await this.submitToAFE();
-      sessionStorage.setItem(
-        sessionStorageKey,
-        JSON.stringify({...sessionEligibilityData, submitted: true})
-      );
+    let {submissionAccepted} = getSessionData();
+    let {submissionSent} = this.state;
```

Comment:
Keeping "sent" in the component state, but "accepted" in the session, prevents us from submitting more than once per page-view, but makes reloading the page try the submission again if it previously failed (which seems like the most user-friendly thing to do here).